provider/pkg/provider/helm_release.go

```diff
@@ -483,7 +580,11 @@ func (r *helmReleaseProvider) helmCreate(ctx context.Context, urn resource.URN,
 	client.WaitForJobs = !newRelease.SkipAwait && newRelease.WaitForJobs
 	client.Devel = newRelease.Devel
 	client.DependencyUpdate = newRelease.DependencyUpdate
-	client.Timeout = time.Duration(newRelease.Timeout) * time.Second
+	timeout := newRelease.Timeout
+	if timeout == 0 {
+		timeout = defaultTimeoutSeconds
+	}
+	client.Timeout = time.Duration(timeout) * time.Second
```

Comment:
```suggestion
	client.Timeout = timeoutOrDefault(newRelease.Timeout)
```

With `timeoutOrDefault` as:
```go
func timeoutOrDefault(timeout int) time.Duration {
	if timeout == 0 {
		timeout = defaultTimeoutSeconds
	}
	return time.Duration(timeout) * time.Second
}